it/connectors/CreateOrAmendExpensesConnectorSpec.scala

```diff
@@ -18,7 +18,8 @@ package connectors
 
 import com.github.tomakehurst.wiremock.http.HttpHeader
 import connectors.httpParsers.CreateOrAmendExpensesHttpParser.CreateOrAmendExpensesResponse
-import models.expenses.CreateExpensesRequestModel
+import models.expenses.createUpdate
+import models.expenses.createUpdate.CreateUpdateExpensesRequest
 import models.{APIErrorBodyModel, APIErrorModel}
```

Comment:
do we need to have both imports ?

Reply:
I can’t see this. Maybe it was already removed by my rebase as there is only one?